insights/formats/__init__.py

```diff
@@ -154,11 +154,20 @@ def get_content(obj, val):
 
 try:
     from jinja2 import Template
+    from jinja2 import exceptions as jinja2Exceptions
 
     def format_rule(comp, val):
         content = get_content(comp, val)
         if content and val.get("type") != "skip":
-            return Template(content).render(val)
+            try:
+                text = Template(content).render(val)
```

Comment:
Would you mind adding such a prefix as below for the content in the `text`?

```
CONTENT:
--------
This system is running on Red Hat Enterprise Linux 7.9

LifeCycle Dates for current system:
...
```

Reply:
Other than this. This PR LGTM.

Reply:
Hi @xiangce ,   
I guess you mean to add this `CONTENT:\n--------` for only the render exception case.    
Updated it accordingly.

Reply:
Fantastic